src/main/resources/org/csanchez/jenkins/plugins/kubernetes/pipeline/scripts/kill-processes-with-cookie.ps1

```diff
@@ -0,0 +1,25 @@
+param(
+    [string]$cookie,
+	[string]$csFile,
+	[string]$killScript
```

Comment:
I guess the kill script cannot just be defined as a function inside this file because of the way it is run with `&`?

Reply:
I was easier to test and develop this way , you have one script that handles the search for processes and another that handles killing process which are both complex matters that you can test and run locally in an isolated way.